sqsdomain/orderbook.go

```diff
@@ -0,0 +1,60 @@
+package sqsdomain
```

Comment:
`sqsdomain` is a separate module that is shared between `sqs` and `osmosis`.

If we do not intend to have the `osmosis` node utilize these APIs, I suggest moving this file into `domain` package instead that is `sqs` specific. This would help reduce dependency management overhead